documentation/general/dash-sonic-hld.md

```diff
@@ -102,8 +103,38 @@ Following are the minimal scaling requirements
 | ACLs per ENI             | 6x10K SRC/DST ports      |
 | CA-PA Mappings           | 10M                      |
 | Active Connections/ENI   | 1M (Bidirectional TCP or UDP)       |
-
-## 1.5 Design Considerations
+| Metering Buckets per ENI | 4000                     |
+
+## 1.5 Metering requirements
+Metering is essential for billing the customers and below are the high-level requirements. Metering/Bucket in this context is related to packet counting for billing purposes and not related to traffic policer or shaping. 
```

Comment:
Instead of "packet counting", we should be "byte counting" while "packet counting" is optional